src/components/organisms/top/top.html

```diff
@@ -0,0 +1,19 @@
+<div class="top">
+  <div class="top-wrap">
+    <img src="../../../img/top.jpg" alt="" />
+    <div class="front-message">
+      <h2 class="advertisement">そうだ。<br />カフェに行こう。</h2>
+    </div>
+  </div>
+  <section class="container">
+    <div class="message">
+      <h2 class="advertisement">
```

Comment:
```suggestion
      <h2>
```
messageの中の `.advertisement` には特にスタイル当ててないと思うので、特別な意図が無い限り混乱を防ぐために削除してしまうのが望ましいです。